### DISCUSSION TITLE
Adding decimal to INT

### DISCUSSION THREAD

**Discussion by jbryan189** [frangoteam/FUXA#1212] [Q&A]
New user here, but I'm needing to make an INT read as a decimal (pseudo float).
I get a number, say 724 and need it to be displayed as 72.4
I know it should be doable, just needing some assistance.

**Comment by unocelli** [comment]
Hi, In modbus you can define a divisor, or via Scale mode
![image](https://github.com/frangoteam/FUXA/assets/48684550/dc946f8a-439b-493e-b724-f82996d7a21c)
